packages/map/src/hooks/useGenerateRuntimeData.ts

```diff
@@ -5,7 +5,23 @@ import { VizFilter } from '@cdc/core/types/VizFilter'
 import { DataRow } from '../types/MapConfig'
 
 const useGenerateRuntimeData = (state: MapConfig) => {
-  const generateRuntimeData = (configObj: MapConfig, filters: VizFilter[], hash: number, test?: boolean) => {
+  /**
+   * Generates runtime data based on the provided configuration object, filters, and hash.
+   *
+   * @param {MapConfig} configObj - The configuration object for the map.
```

Comment:
I wouldn't add all these comments for each param because they could change in the future and you'd need to update them in two places. I think the typescript itself is self documenting enough here.